test/stresstest/broker.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
```

Comment:
No license header

Reply:
Is there a way to auto generate and add them? Or just manually copy them in?